talisker/util.py

```diff
@@ -22,6 +22,9 @@
 import pip
 from builtins import *  # noqa
 
+import functools
+
+import werkzeug.local
```

Comment:
The comment here was really just a counter to my, "This is good - celery shouldn't really depend on anything HTTP'y", comment above.

Although I always thought werkzeug was specifically an HTTP/WSGI utility library.

Not that it really matters - I agree it's beyond this MP.